pkg/tagger/collectors/docker_main.go

```diff
@@ -89,11 +89,11 @@ func (c *DockerCollector) Stop() error {
 
 // Fetch inspect a given container to get its tags on-demand (cache miss)
 func (c *DockerCollector) Fetch(container string) ([]string, []string, error) {
-	cid := strings.TrimPrefix(container, docker.DockerEntityPrefix)
-	if cid == container || len(cid) == 0 {
+	cID := strings.TrimPrefix(container, docker.DockerEntityPrefix)
+	if cID == container || len(cID) == 0 {
 		return nil, nil, ErrNotFound
```

Comment:
let's return `nil, nil, nil` here so that the tagger does not query DockerCollector again for every `kubernetes_pod://` request

Reply:
oooh that's nice